src/jailer/src/cgroup.rs

```diff
@@ -288,10 +288,13 @@ fn inherit_from_parent(path: &mut PathBuf, file_name: &str, depth: u16) -> Resul
 // Extract the controller name from the cgroup file. The cgroup file must follow
 // this format: <cgroup_controller>.<cgroup_property>.
 fn get_controller_from_filename(file: &str) -> Result<&str> {
+    if file.contains('/') {
```

Comment:
why do we specifically perform this check?
There are other characters which are not allowed in the controller name as well, right?

Reply:
As discussed offline, this line is not necessary. Removed.